### DISCUSSION TITLE
The only user SO console superuser role revoked

### DISCUSSION THREAD

**Discussion by UMWP** [Security-Onion-Solutions/securityonion#11603] [General] [answered]
Hello.

It's on prem 2.4.20 distributed installation and I've mistakenly revoked my only GUI user superuser role in SOC menu: Administration->Users . I can't roll it back via that menu (I can't assign superuser again). Howewer if I type on management node CLI:
sudo so-user list
it still shows that user as superuser

Is there any way to fix it?

Regards
UMWP

**Comment by cm-ops** [comment]
Did you disable the user? If so, have you tried to enable the user with `so-user`? If not, have you tried creating a new user with `superuser` role and assigning changing the role for you user?

**Comment by UMWP** [comment, marked as answer]
Thank You very much. I have created a brand new user with superuser role as You have mentioned.

Regards
UMWP
